tests/unit/h/services/group_members_test.py

```diff
@@ -71,7 +71,7 @@ def test_it_adds_users_in_userids(self, factories, group_members_service):
 
         group_members_service.add_members(group, userids)
 
-        assert group.members == users
+        assert all(user in group.members for user in users)
```

Comment:
To clarify, the expected member set here is `{ group.creator, *users }`?

Reply:
Yeah, but `group.creator` is irrelevant to this particular test which only cares the `users` were added